### ISSUE TITLE
AtlasTileMapLoader crashes when using tile objects

### ISSUE THREAD

**Issue by dallen559** [libgdx/libgdx#3883]
#### Issue details

When using AtlasTileMapLoader to load generated TiledMaps with atlas tilesets, the maps containing TileMapObjects fail to load. Removing all TileMapObjects and replacing them with generic Rectangle map objects work fine. The map is saved using a single tileset with multiple tile layers and object layers. These maps are saved as base64 compressed maps.

Using the latest release of Tiled. 
#### Reproduction steps/code

-Create a new map (no specific preferences)
-Add an object layer and place a tile map object
-Save and generate the atlas and packed tileset with the Libgdx TiledMapPacker tool
-Load the .tmx using an AtlasTileMapLoader

Following the same steps without placing tile map objects loads without issues.
#### Version of LibGDX and/or relevant dependencies

Latest version of LibGDX with Tools included
Tiled v0.15.1
#### Stacktrace

``` java
``
Exception in thread "LWJGL Application" com.badlogic.gdx.utils.GdxRuntimeException: com.badlogic.gdx.utils.GdxRuntimeException: Couldn't load asset: map/level1.tmx
    at com.badlogic.gdx.assets.AssetManager.handleTaskError(AssetManager.java:570)
    at com.badlogic.gdx.assets.AssetManager.update(AssetManager.java:375)
    at com.compulsive.engine.LoadingScreen.render(LoadingScreen.java:55)
    at com.badlogic.gdx.Game.render(Game.java:46)
    at com.badlogic.gdx.backends.lwjgl.LwjglApplication.mainLoop(LwjglApplication.java:218)
    at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:123)
Caused by: com.badlogic.gdx.utils.GdxRuntimeException: Couldn't load asset: map/level1.tmx
    at com.badlogic.gdx.assets.AssetLoadingTask.handleAsyncLoader(AssetLoadingTask.java:139)
    at com.badlogic.gdx.assets.AssetLoadingTask.update(AssetLoadingTask.java:90)
    at com.badlogic.gdx.assets.AssetManager.updateTask(AssetManager.java:498)
    at com.badlogic.gdx.assets.AssetManager.update(AssetManager.java:373)
    ... 4 more
Caused by: com.badlogic.gdx.utils.GdxRuntimeException: com.badlogic.gdx.utils.GdxRuntimeException: Couldn't load tilemap 'map/level1.tmx'
    at com.badlogic.gdx.utils.async.AsyncResult.get(AsyncResult.java:46)
    at com.badlogic.gdx.assets.AssetLoadingTask.handleAsyncLoader(AssetLoadingTask.java:137)
    ... 7 more
Caused by: com.badlogic.gdx.utils.GdxRuntimeException: Couldn't load tilemap 'map/level1.tmx'
    at com.badlogic.gdx.maps.tiled.AtlasTmxMapLoader.loadAsync(AtlasTmxMapLoader.java:217)
    at com.badlogic.gdx.maps.tiled.AtlasTmxMapLoader.loadAsync(AtlasTmxMapLoader.java:61)
    at com.badlogic.gdx.assets.AssetLoadingTask.call(AssetLoadingTask.java:74)
    at com.badlogic.gdx.assets.AssetLoadingTask.call(AssetLoadingTask.java:34)
    at com.badlogic.gdx.utils.async.AsyncExecutor$2.call(AsyncExecutor.java:58)
    at java.util.concurrent.FutureTask.run(FutureTask.java:262)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
    at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NullPointerException
    at com.badlogic.gdx.maps.tiled.objects.TiledMapTileMapObject.<init>(TiledMapTileMapObject.java:42)
    at com.badlogic.gdx.maps.tiled.BaseTmxMapLoader.loadObject(BaseTmxMapLoader.java:217)
    at com.badlogic.gdx.maps.tiled.BaseTmxMapLoader.loadObjectGroup(BaseTmxMapLoader.java:120)
    at com.badlogic.gdx.maps.tiled.AtlasTmxMapLoader.loadMap(AtlasTmxMapLoader.java:276)
    at com.badlogic.gdx.maps.tiled.AtlasTmxMapLoader.loadAsync(AtlasTmxMapLoader.java:215)
    ... 8 more
```
#### Please select the affected platforms
- [ ] Android
- [ ] iOS
- [ ] HTML/GWT
- [x] Windows
- [ ] Linux
- [ ] MacOS

**Comment by saltares** [comment]
This is probably because the map is missing the atlas property.

**Comment by tianqiujie** [comment]
I also have the issues

**Comment by mgsx-dev** [comment]
there was a big refactoring and a lot of fixes for TMX loaders in libGDX 1.9.9, 1.9.10 and 1.9.11.
considering how old is this issue, Tiled version used and changes made since then, i'll close the issue.
